lib/sidekiq.rb

```diff
@@ -127,9 +129,13 @@ def self.logger=(log)
     Sidekiq::Logging.logger = log
   end
 
+  # When set, overrides Sidekiq.options[:global_poll_interval_average] and sets the
+  # average interval that this process will delay before checking for scheduled jobs
+  # or job retries that are ready to run.
+  #
   # See sidekiq/scheduled.rb for an in-depth explanation of this value
   def self.poll_interval=(interval)
```

Comment:
Provide a setter for the new attribute.  I don't like people accessing Sidekiq.options directly and never document it.